scram/route_manager/tests/test_views.py

```diff
@@ -13,3 +14,45 @@ def test_root_url_resolves_to_home_page_view(self):
         """Ensure we can find the home page."""
         found = resolve("/")
         self.assertEqual(found.func, home_page)
+
+
+class HomePageFirstVisitTest(TestCase):
+    """Test how the home page renders the first time we view it."""
+
+    def setUp(self):
+        """Get the home page."""
+        self.response = self.client.get(reverse("route_manager:home"))
+
+    def test_first_homepage_view_has_userinfo(self):
```

Comment:
Won't this depend if auto create admin is true?

Reply:
Yes, but I think right now, our test suite makes several assumptions about the configuration (i.e. we don't read `AUTOCREATE_ADMIN` from an env var, so we would need to manually modify config/settings/test.py and re-run the test). We should figure out how to test various setting combinations.